### PR TITLE
Translate overflowing content

### PR DESCRIPTION
**Pull request by VDV186**
Если хотите что-нибудь уточнить или знаете, как сделать перевод точнее, я не возражаю. :-)

### CODE REVIEW

**Comment by VDV186** [comment]
Александр, как label ставить pull request?

**Comment by myshov** [comment]
@VDV186 лейблы с языком перевода ставят менйтейнеры репозитория, но насколько я знаю, это хотят автоматизировать в будущем

File: files/ru/learn/css/building_blocks/overflowing_content/index.html [hunk 1]
```diff
@@ -26,71 +26,77 @@ <h2 id="Что_такое_overflow">Что такое overflow?</h2>
...
+<p><strong>Примечание</strong>: вы можете точно определить прокрутку по осям x и y, используя свойство <code>overflow</code>, передавая два значения. Если указаны два ключевых параметра, первый относится к <code>overflow-x</code>, а второй – к <code>overflow-y</code>. Иначе оба <code>overflow-x</code> и <code>overflow-y</code> иметь одно и то же значение. Например, <code>overflow: scroll hidden</code> устанавливает <code>overflow-x</code> на <code>scroll</code> и <code>overflow-y</code> на <code>hidden</code>.</p>
 </div>
 
-<p>If you only want scrollbars to appear if there is more content than can fit in the box, then use <code>overflow: auto</code>. In this case it is left up to the browser to decide whether to display scrollbars. Desktop browsers will typically only do so once there is enough content to cause overflow.</p>
+<p>Если вы хотите, чтобы полосы прокрутки отображались только тогда, когда содержимого больше, чем может поместиться в поле, используйте <code>overflow: auto</code>. Это позволяет браузеру определять, следует ли отображать полосы прокрутки.</p>
```
Comment (Saionaro) [inline files/ru/learn/css/building_blocks/overflowing_content/index.html:75]:
```suggestion
<p>Если вы хотите, чтобы полосы прокрутки отображались только тогда, когда содержимого больше, чем может поместиться в поле, используйте <code>overflow: auto</code>. Это позволяет браузеру автоматически определять, следует ли отображать полосы прокрутки.</p>
```

File: files/ru/learn/css/building_blocks/overflowing_content/index.html [hunk 2]
```diff
@@ -26,71 +26,77 @@ <h2 id="Что_такое_overflow">Что такое overflow?</h2>
...
 
-<p><strong>In the below example, remove some of the content until it fits into the box and you should see the scrollbars disappear.</strong></p>
+<p><strong>В приведенном ниже примере уменьшите количество содержимого, так чтобы оно поместилось в поле. Вы должны увидеть, что полосы прокрутки исчезнут.</strong></p>
```
Comment (Saionaro) [inline files/ru/learn/css/building_blocks/overflowing_content/index.html:77]:
```suggestion
<p><strong>В приведенном ниже примере уменьшите количество содержимого так, чтобы оно поместилось в поле. Вы должны увидеть, что полосы прокрутки исчезнут.</strong></p>
```

File: files/ru/learn/css/building_blocks/overflowing_content/index.html [hunk 3]
```diff
@@ -26,71 +26,77 @@ <h2 id="Что_такое_overflow">Что такое overflow?</h2>
...
 
-<h2 id="Unwanted_overflow_in_web_design">Unwanted overflow in web design</h2>
+<h2 id="Unwanted_overflow_in_web_design">Нежелательный overflow в веб-разработке</h2>
 
-<p>Modern layout methods (as covered in the <a href="/en-US/docs/Learn/CSS/CSS_layout">CSS layout</a> module) manage overflow very well. They have been designed to cope with the fact that we tend not to be able to predict how much content we have on the web. In the past however, developers would often use fixed heights to try to line up the bottoms of boxes that really had no relationship to each other. This was fragile, and in a legacy application you may occasionally come across a box where the content is overlaying other content on the page. If you see this you now know that what is happening is overflow; ideally you would refactor the layout to not rely on fixing the box size.</p>
+<p>Современные методы раскладки (описанные в разделе <a href="/en-US/docs/Learn/CSS/CSS_layout">CSS раскладка</a>) справляются с overflow очень хорошо. Они хорошо работают в независимости от того, сколько контента будет на веб-странице.</p>
```
Comment (Saionaro) [inline files/ru/learn/css/building_blocks/overflowing_content/index.html:87]:
```suggestion
<p>Современные методы раскладки (описанные в разделе <a href="/en-US/docs/Learn/CSS/CSS_layout">CSS раскладка</a>) справляются с перекрытием очень хорошо вне зависимости от того, сколько контента будет на веб-странице.</p>
```

File: files/ru/learn/css/building_blocks/overflowing_content/index.html [hunk 4]
```diff
@@ -26,71 +26,77 @@ <h2 id="Что_такое_overflow">Что такое overflow?</h2>
...
+<p>Это не всегда было нормой. В прошлом некоторые сайты были построены с блоками фиксированной высоты для выравнивания нижних границ блоков. Тем не менее эти блоки могли не иметь ничего общего между собой. Это была хрупкая конструкция. В устаревших приложениях вы можете встретить блок, в котором содержимое перекрывает другое содержимое на странице. Теперь вы понимаете, что это происходит из-за overflow. В идеале вы должны провести рефакторинг разметки, чтобы не полагаться на блоки с фиксированной высотой.</p>
 
-<h2 id="Summary">Summary</h2>
+<p>При разработке сайта всегда помните о перекрытии. Тестируйте дизайны с большим и малым количеством контента. Увеличивайте размер шрифта текста. Убедитесь, что ваш CSS работает надежно. Изменение значения overflow для скрытия содержимого или добавления полос прокрутки, должно использоваться только по необходимости (например, там, где вы хотите использовать поле прокрутки).</p>
```
Comment (Saionaro) [inline files/ru/learn/css/building_blocks/overflowing_content/index.html:91]:
```suggestion
<p>При разработке сайта всегда помните о перекрытии. Тестируйте дизайны как с большим, так и с малым количеством контента. Проверяйте различные размеры шрифта текстов. Убедитесь, что ваш CSS работает надежно. Изменение значения overflow для скрытия содержимого или добавления полос прокрутки, должно использоваться только при необходимости (например там, где вы хотите использовать прокручиваемый блок).</p>
```

File: files/ru/learn/css/building_blocks/overflowing_content/index.html [hunk 5]
```diff
@@ -26,71 +26,77 @@ <h2 id="Что_такое_overflow">Что такое overflow?</h2>
...
 
-<p>This short lesson has introduced the concept of overflow; you now understand that CSS tries not to make overflowing content invisible as this will cause data loss. You have discovered that you can manage potential overflow, and also that you should test your work to make sure you do not accidentally cause problematic overflow.</p>
+<h2 id="Test_your_skills!">Проверьте свои навыки!</h2>
+
+<p>В этой статье мы рассмотрели многое, но можете ли вы вспомнить самую важную информацию? Вы можете найти дополнительные тесты, чтобы убедиться, что вы усвоили эту информацию, прежде чем двигаться дальше – см. <a href="/en-US/docs/Learn/CSS/Building_blocks/Overflow_Tasks">Проверь свои навыки: overflow</a>.</p>
+
+<h2 id="Summary">Заключение</h2>
+
+<p>В этом уроке была представлена концепция overflow. Вы должны понимать, что CSS по умолчанию позволяет избежать невидимости содержимого выходящего за границы блока. Вы обнаружили, что можете справиться с возникшим перекрытием, а также что вам следует тестировать работу, чтобы быть уверенным, что в ней случайно не возникнет перекрытия.</p>
```
Comment (Saionaro) [inline files/ru/learn/css/building_blocks/overflowing_content/index.html:99]:
```suggestion
<p>В этом уроке была представлена концепция перекрытия. Важно понимать, что CSS по умолчанию старается избежать обрезания выходящего за границы блока содержимого. Мы изучили, как можно справиться с возникшим перекрытием, а также рассмотрели важность тестирования поведения веб-страниц как с малым количеством контента, так и с большим.</p>
```

**Review by saionaro — COMMENTED** [review]
@VDV186 привет! Спасибо за перевод!

Я оставил несколько предложений к формулировкам - посмотрите и примените те, с которыми будете согласны. К прочим - комментарии и обсуждение приветствуется.

Пожалуйста, проверьте ссылки на MDN - если есть статьи в русском разделе, то следует поменять английские ссылки на них (`en-Us => ru`).

У ссылок, которые ведут на англоязычные статьи стоит добавить пометку `(EN)` [вот так](https://github.com/mdn/translated-content/blob/main/files/ru/learn/accessibility/css_and_javascript/index.html#L194)

Также, пожалуйста, проверьте, что везде. где необходимо используется буква "ё" - вроде бы я её встречал реже, чем обычно, однако нигде точно глаз не зацепился.

File: files/ru/learn/css/building_blocks/overflowing_content/index.html [hunk 6]
```diff
@@ -26,71 +26,77 @@ <h2 id="Что_такое_overflow">Что такое overflow?</h2>
...
 
-<p>The  {{cssxref("overflow")}} property is how you take control of an element's overflow and tell the browser how you want it to behave. The default value of overflow is <code>visible</code>, which is why by default we can see our content when it overflows.</p>
+<p>Свойство {{cssxref("overflow")}} позволяет взять под контроль перекрытие элемента и подсказать браузеру, как он должен себя вести. Значение overflow по умолчанию – <code>visible</code>, что означает - «показывать контент, когда он выходит за границы блока».</p>
 
-<p>If you want to crop the content when it overflows you can set <code>overflow: hidden</code> on your box. This will do exactly what it says — hide the overflow. This may well cause things to vanish so you should only ever do this if hiding content is not going to cause a problem.</p>
+<p>Чтобы обрезать контент выходящий за пределы блока, вы можете установить <code>overflow: hidden</code>. Это делает именно то, о чем говорит: скрывает выходящий за пределы контент. Помните, что это может сделать часть содержимого невидимым. Используйте данное значение только в том случае, если скрытие содержимого не вызовет проблем.</p>
```
Comment (myshov) [inline files/ru/learn/css/building_blocks/overflowing_content/index.html:51]:
```suggestion
<p>Чтобы обрезать контент выходящий за пределы блока, вы можете установить <code>overflow: hidden</code>. Это свойство делает именно то, о чём говорит: скрывает выходящий за пределы контент. Помните, что это может сделать часть содержимого невидимым. Используйте данное значение только в том случае, если скрытие содержимого не вызовет проблем.</p>
```

File: files/ru/learn/css/building_blocks/overflowing_content/index.html [hunk 7]
```diff
@@ -26,71 +26,77 @@ <h2 id="Что_такое_overflow">Что такое overflow?</h2>
...
 
 <p>{{EmbedGHLiveSample("css-examples/learn/overflow/hidden.html", '100%', 600)}}</p>
 
-<p>Perhaps you would instead like to add scrollbars when content overflows? If you use <code>overflow: scroll</code> then your browser will always display scrollbars — even if there is not enough content to overflow. You may want this, as it prevents scrollbars appearing and disappearing depending on content.</p>
+<p>Возможно, вместо этого, вы захотите добавить полосы прокрутки при возникновении перекрытия. При использовании <code>overflow: scroll</code> браузеры с видимыми полосами прокрутки всегда будут отображать их, даже если содержимого недостаточно для возникновения перекрытия. Это дает возможность сохранения целостности раскладки вместо появления или исчезновения полос прокрутки в зависимости от количества содержимого в контейнере.</p>
```
Comment (myshov) [inline files/ru/learn/css/building_blocks/overflowing_content/index.html:55]:
```suggestion
<p>Возможно, что при возникновении перекрытия вместо скрытия вы захотите отобразить полосы прокрутки. При использовании <code>overflow: scroll</code> браузеры с видимыми полосами прокрутки всегда будут отображать их, даже если содержимого недостаточно для возникновения перекрытия. Это позволяет сохранить целостность раскладки, так как полосы прокрутки не будут появляться и пропадать в зависимости от количества содержимого в контейнере.</p>
```

File: files/ru/learn/css/building_blocks/overflowing_content/index.html [hunk 8]
```diff
@@ -26,71 +26,77 @@ <h2 id="Что_такое_overflow">Что такое overflow?</h2>
...
 
-<p><strong>If you remove some of the content from the box below, you'll see that the scrollbars still remain even with nothing to scroll (or at least just the scrollbar tracks).</strong></p>
+<p><strong>Удалите часть содержимого из поля ниже. Обратите внимание, что полосы прокрутки остаются, даже если прокрутка не требуется.</strong></p>
 
 <p>{{EmbedGHLiveSample("css-examples/learn/overflow/scroll.html", '100%', 600)}}</p>
 
-<p>In the above example we only need to scroll on the <code>y</code> axis, however we get scrollbars in both axes. You could instead use the {{cssxref("overflow-y")}} property, setting <code>overflow-y: scroll</code> to only scroll on the <code>y</code> axis.</p>
+<p>В приведенном выше примере нам нужно прокручивать содержимое только по оси <code>y</code>, однако мы получаем полосы прокрутки по обеим осям. Вы можете использовать свойство {{cssxref("overflow-y")}}, которое позволяет прокручивать содержимое только по оси <code>y</code>.</p>
```
Comment (myshov) [inline files/ru/learn/css/building_blocks/overflowing_content/index.html:61]:
```suggestion
<p>В приведённом выше примере нам нужно прокручивать содержимое только по оси <code>y</code>, однако мы получаем полосы прокрутки по обеим осям. Вы можете использовать свойство {{cssxref("overflow-y")}}, которое позволяет прокручивать содержимое только по оси <code>y</code>.</p>
```

File: files/ru/learn/css/building_blocks/overflowing_content/index.html [hunk 9]
```diff
@@ -26,71 +26,77 @@ <h2 id="Что_такое_overflow">Что такое overflow?</h2>
...
 
 <p>{{EmbedGHLiveSample("css-examples/learn/overflow/scroll-y.html", '100%', 600)}}</p>
 
-<p>You could also scroll on the x axis using {{cssxref("overflow-x")}}, although this is not a recommended way to deal with long words! If you do need to deal with a long word in a small box then you could look at the {{cssxref("word-break")}} or {{cssxref("overflow-wrap")}} properties. In addition some of the methods discussed in the <a href="/en-US/docs/Learn/CSS/Building_blocks/Sizing_items_in_CSS">Sizing items in CSS</a> lesson may help you to create boxes that cope better with varying amounts of content.</p>
+<p>Вы также можете установить прокрутку по оси x с помощью {{cssxref("overflow-x")}}, хотя это не рекомендуемый способ для размещения длинных слов! Если у вас есть длинное слово в маленьком поле, вы можете использовать свойства {{cssxref("word-break")}} или {{cssxref("overflow-wrap")}}. Кроме того, некоторые методы, описанные в разделе <a href="/en-US/docs/Learn/CSS/Building_blocks/Sizing_items_in_CSS">Изменение размеров в CSS</a> могут помочь вам создавать блоки, которые лучше масштабируются с различным объемом содержимого.</p>
```
Comment (myshov) [inline files/ru/learn/css/building_blocks/overflowing_content/index.html:65]:
```suggestion
<p>Вы также можете установить прокрутку по оси x с помощью {{cssxref("overflow-x")}}, но это не рекомендуемый способ отображения длинных слов! Если у вас есть длинное слово в маленьком поле, вы можете использовать свойства {{cssxref("word-break")}} или {{cssxref("overflow-wrap")}}. Кроме того, некоторые методы, описанные в разделе <a href="/en-US/docs/Learn/CSS/Building_blocks/Sizing_items_in_CSS">Изменение размеров в CSS</a>, могут помочь вам создавать блоки, которые лучше масштабируются с различным объемом содержимого.</p>
```

File: files/ru/learn/css/building_blocks/overflowing_content/index.html [hunk 10]
```diff
@@ -26,71 +26,77 @@ <h2 id="Что_такое_overflow">Что такое overflow?</h2>
...
 
 <div class="blockIndicator note">
-<p><strong>Note</strong>: that you can specify x and y scrolling using the <code>overflow</code> property and passing in two values. If two keywords are specified, the first applies to <code>overflow-x</code> and the second to <code>overflow-y</code>. Otherwise, both <code>overflow-x</code> and <code>overflow-y</code> are set to the same value. For example, <code>overflow: scroll hidden</code> would set <code>overflow-x</code> to <code>scroll</code> and <code>overflow-y</code> to <code>hidden</code>.</p>
+<p><strong>Примечание</strong>: вы можете точно определить прокрутку по осям x и y, используя свойство <code>overflow</code>, передавая два значения. Если указаны два ключевых параметра, первый относится к <code>overflow-x</code>, а второй — к <code>overflow-y</code>. Иначе оба <code>overflow-x</code> и <code>overflow-y</code> будут иметь одно и то же значение. Например, <code>overflow: scroll hidden</code> устанавливает <code>overflow-x</code> на <code>scroll</code> и <code>overflow-y</code> на <code>hidden</code>.</p>
```
Comment (myshov) [inline files/ru/learn/css/building_blocks/overflowing_content/index.html:72]:
```suggestion
<p><strong>Примечание</strong>: вы можете точно задать прокрутку по осям x и y, используя свойство <code>overflow</code>, передав два значения. Первое значение будет относиться к <code>overflow-x</code>, а второе — к <code>overflow-y</code>. Если передать одно значение, то <code>overflow-x</code> и <code>overflow-y</code> получат одинаковые значения. Например, <code>overflow: scroll hidden</code> устанавливает <code>overflow-x</code> в <code>scroll</code> и <code>overflow-y</code> в <code>hidden</code>.</p>
```

File: files/ru/learn/css/building_blocks/overflowing_content/index.html [hunk 11]
```diff
@@ -26,71 +26,77 @@ <h2 id="Что_такое_overflow">Что такое overflow?</h2>
...
 </div>
 
-<p>If you only want scrollbars to appear if there is more content than can fit in the box, then use <code>overflow: auto</code>. In this case it is left up to the browser to decide whether to display scrollbars. Desktop browsers will typically only do so once there is enough content to cause overflow.</p>
+<p>Если вы хотите, чтобы полосы прокрутки отображались только тогда, когда содержимого больше, чем может поместиться в поле, используйте <code>overflow: auto</code>. Это позволяет браузеру автоматически определять, следует ли отображать полосы прокрутки.</p>
 
-<p><strong>In the below example, remove some of the content until it fits into the box and you should see the scrollbars disappear.</strong></p>
+<p><strong>В приведенном ниже примере уменьшите количество содержимого так, чтобы оно поместилось в поле. Вы должны увидеть, что полосы прокрутки исчезнут.</strong></p>
```
Comment (myshov) [inline files/ru/learn/css/building_blocks/overflowing_content/index.html:77]:
```suggestion
<p><strong>В приведённом ниже примере уменьшите количество содержимого так, чтобы оно поместилось в поле. Вы должны увидеть, что полосы прокрутки исчезнут.</strong></p>
```

File: files/ru/learn/css/building_blocks/overflowing_content/index.html [hunk 12]
```diff
@@ -26,71 +26,77 @@ <h2 id="Что_такое_overflow">Что такое overflow?</h2>
...
+<p>Это не всегда было нормой. В прошлом некоторые сайты были построены с блоками фиксированной высоты для выравнивания нижних границ блоков. Тем не менее эти блоки могли не иметь ничего общего между собой. Это была хрупкая конструкция. В устаревших приложениях вы можете встретить блок, в котором содержимое перекрывает другое содержимое на странице. Теперь вы понимаете, что это происходит из-за overflow. В идеале вы должны провести рефакторинг разметки, чтобы не полагаться на блоки с фиксированной высотой.</p>
 
-<h2 id="Summary">Summary</h2>
+<p>При разработке сайта всегда помните о перекрытии. Тестируйте дизайны как с большим, так и с малым количеством контента. Проверяйте различные размеры шрифта текстов. Убедитесь, что ваш CSS работает надежно. Изменение значения overflow для скрытия содержимого или добавления полос прокрутки, должно использоваться только при необходимости (например там, где вы хотите использовать прокручиваемый блок).</p>
```
Comment (myshov) [inline files/ru/learn/css/building_blocks/overflowing_content/index.html:91]:
```suggestion
<p>При разработке сайта всегда помните о перекрытии. Тестируйте дизайны как с большим, так и с малым количеством контента. Проверяйте различные размеры шрифта текстов. Убедитесь, что ваш CSS работает надёжно. Изменение значения overflow для скрытия содержимого или добавления полос прокрутки, должно использоваться только при необходимости (например там, где вы хотите использовать прокручиваемый блок).</p>
```

**Review by myshov — COMMENTED** [review]
Спасибо за перевод! 🔥
В целом, всё хорошо, но нашёл пару моментов, которые хотелось бы исправить.

**Comment by myshov** [comment]
@VDV186 спасибо!

**Comment by VDV186** [comment]
Всем большое спасибо.
